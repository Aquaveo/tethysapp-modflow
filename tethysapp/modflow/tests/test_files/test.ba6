# BAS6 package for  MODFLOW-NWT, generated by Flopy.
FREE
INTERNAL               1     (FREE) -1 #ibound_layer_0                
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
    -999.0
INTERNAL               1     (FREE) -1 #strt_layer_0                  
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
   9.500000E+00   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   1.000000E+01   9.000000E+00
