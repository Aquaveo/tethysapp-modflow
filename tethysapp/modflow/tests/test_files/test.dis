# DIS package for  MODFLOW-NWT, generated by Flopy.
         1        10        10         1         4         2
  0
CONSTANT    5.490000E+02                           #delr                          
CONSTANT    3.030000E+02                           #delc                          
CONSTANT    1.000000E+01                           #model_top                     
INTERNAL               1     (FREE) -1 #botm_layer_0                  
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00
      1.000000             1  1.000000  SS 
