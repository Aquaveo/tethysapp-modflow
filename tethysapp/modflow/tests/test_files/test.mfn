LIST                2         test.out   #  1. Ftype Nunit Fname [Option]
DATA(BINARY)        3         test.hds   #  1. Ftype Nunit Fname [Option]
DATA(BINARY)        40        test.cbb   #  1. Ftype Nunit Fname [Option]
DIS                 7         test.dis   #  1. Ftype Nunit Fname [Option]
BAS6                8         test.ba6   #  1. Ftype Nunit Fname [Option]
UPW                 12        test.upw   #  1. Ftype Nunit Fname [Option]
OC                  10        test.oc    #  1. Ftype Nunit Fname [Option]
SFR                 14        test.sfr   #  1. Ftype Nunit Fname [Option]
NWT                 13        test.nwt   #  1. Ftype Nunit Fname [Option]
