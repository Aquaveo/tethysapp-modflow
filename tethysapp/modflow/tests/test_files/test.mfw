ORIG 439760.0 4455320.0 10.0
ROTZ 90.0
PRJ_FILE "test.prj"
