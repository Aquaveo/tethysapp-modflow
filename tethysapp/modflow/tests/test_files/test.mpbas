# # MPBAS package for MODPATH 7, generated by Flopy.
-999 -888
0                   # DEFAULTIFACECOUNT
         1
INTERNAL               1    (10I10) -1 #ibound layer 1                
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
        -1         1         1         1         1         1         1         1         1        -1
CONSTANT    3.000000E-01                           #porosity layer 1              
