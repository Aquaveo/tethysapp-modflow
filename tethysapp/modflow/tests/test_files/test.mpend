MODPATH_ENDPOINT_FILE         7         2
    2         1         1         1   0.0000000000E+00  0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
       0       0       0       0       1       0       0       0       0       0
         1
flow_path       
END HEADER
         1         1         1    4  0.0000000000E+00  0.6102776654E+06        48    1  0.6867099000E+00  0.3801875000E+00  0.9160817000E+00  0.4220003735E+04  0.1630196812E+04  0.8392056945E+01    1    0        46    1  0.1000000000E+01  0.4633076388E+00  0.9160817000E+00  0.3294000000E+04  0.1655382215E+04  0.8546406179E+01    1    2
