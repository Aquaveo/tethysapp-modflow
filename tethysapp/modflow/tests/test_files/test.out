                                  MODFLOW-NWT-SWR1 
      U.S. GEOLOGICAL SURVEY MODULAR FINITE-DIFFERENCE GROUNDWATER-FLOW MODEL
                             VERSION 1.1.4 4/01/2018
                    BASED ON MODFLOW-2005 VERSION 1.12.00 02/03/2017

 LIST FILE: test.out
                         UNIT    2

 OPENING test.dis
 FILE TYPE:DIS   UNIT    7   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.ba6
 FILE TYPE:BAS6   UNIT    8   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.upw
 FILE TYPE:UPW   UNIT   12   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.oc
 FILE TYPE:OC   UNIT   10   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.sfr
 FILE TYPE:SFR   UNIT   14   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.nwt
 FILE TYPE:NWT   UNIT   13   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.wel
 FILE TYPE:WEL   UNIT   20   STATUS:OLD    
 FORMAT:FORMATTED              ACCESS:SEQUENTIAL          

 OPENING test.cbb
 FILE TYPE:DATA(BINARY)   UNIT   40   STATUS:REPLACE
 FORMAT:UNFORMATTED            ACCESS:STREAM              

 OPENING test.hds
 FILE TYPE:DATA(BINARY)   UNIT    3   STATUS:REPLACE
 FORMAT:UNFORMATTED            ACCESS:STREAM              

 BAS -- BASIC PACKAGE, VERSION 1.1.4, 4/01/2018 INPUT READ FROM UNIT    8

 MODFLOW was compiled using mixed precision
 Precision of REAL variables:           6
 Precision of DOUBLE PRECISION variables:          15

 DISCRETIZATION INPUT DATA READ FROM UNIT    7
 # DIS package for  MODFLOW-NWT, generated by Flopy.
    1 LAYERS        10 ROWS        10 COLUMNS
    1 STRESS PERIOD(S) IN SIMULATION
 MODEL TIME UNIT IS DAYS
 MODEL LENGTH UNIT IS METERS
  Confining bed flag for each layer:
   0

                     DELR =   549.000    

                     DELC =   303.000    

 TOP ELEVATION OF LAYER 1 =   10.0000    



             MODEL LAYER BOTTOM EL. FOR LAYER   1
 READING ON UNIT    7 WITH FORMAT: (FREE)              


 STRESS PERIOD     LENGTH       TIME STEPS     MULTIPLIER FOR DELT    SS FLAG
 ----------------------------------------------------------------------------
        1         1.000000          1                    1.000         SS

 STEADY-STATE SIMULATION




 # BAS6 package for  MODFLOW-NWT, generated by Flopy.                            
 THE FREE FORMAT OPTION HAS BEEN SELECTED



                     BOUNDARY ARRAY FOR LAYER   1
 READING ON UNIT    8 WITH FORMAT: (FREE)              

 AQUIFER HEAD WILL BE SET TO  -999.00     AT ALL NO-FLOW NODES (IBOUND=0).



                       INITIAL HEAD FOR LAYER   1
 READING ON UNIT    8 WITH FORMAT: (FREE)              
 # OC package for  MODFLOW-NWT, generated by Flopy.

 OUTPUT CONTROL IS SPECIFIED ONLY AT TIME STEPS FOR WHICH OUTPUT IS DESIRED
 HEAD PRINT FORMAT CODE IS   0    DRAWDOWN PRINT FORMAT CODE IS   0
 HEADS WILL BE SAVED ON UNIT    3    DRAWDOWNS WILL BE SAVED ON UNIT    0
 # NWT package for  MODFLOW-NWT, generated by Flopy.

 NWT1 -- Newton Solver, VERSION 1.1.4, 4/01/2018
         INPUT READ FROM UNIT 13

 SIMPLE OPTION:
 DEFAULT SOLVER INPUT VALUES REFLECT NEARLY LINEAR MODEL
 ***GMRES linear solver will be used***

   CONVERGENCE CRITERION OF   0.100000E-03 FOR HEAD SOLUTION
   AND A TOLERANCE OF   0.500000E+03 FOR FLOW SOLUTION AND 
   A MAXIMUM OF   400 OUTER ITERATIONS. 


   D-B-D REDUCTION FACTOR OF    0.970000E+00 AND 
   A D-B-D INCREASE FACTOR OF    0.100000E-03 AND 
   A D-B-D RELAXATION OF    0.000000E+00 AND 
   A MOMENTUM FACTOR OF    0.000000E+00 .


 ***BACKTRACKING IS INACTIVE***
 GMRES -- LINEAR SOLUTION BY GMRES PACKAGE 
     adapted for HYDROTHERM by Kipp and others (2008)
     using software originally developed by Saad (1990) 13
     using algorithms described in Saad (2003)

 UPW1 -- UPSTREAM WEIGHTING FLOW PACKAGE, VERSION 1.1.4, 4/01/2018
         INPUT READ FROM UNIT 12

 # UPW package for  MODFLOW-NWT, generated by Flopy.
 CELL-BY-CELL FLOWS WILL BE SAVED ON UNIT   40
 No named parameters

   LAYER FLAGS:
 LAYER       LAYTYP          LAYAVG    CHANI           LAYVKA           LAYWET
 ---------------------------------------------------------------------------
    1             1             0    -1.000E+00             1             0

   INTERPRETATION OF LAYER FLAGS:
                        INTERBLOCK     HORIZONTAL    DATA IN
         LAYER TYPE   TRANSMISSIVITY   ANISOTROPY   ARRAY VKA   WETTABILITY
 LAYER   (LAYTYP)        (LAYAVG)      (CHANI)     (LAYVKA)       (LAYWET)
 ---------------------------------------------------------------------------
    1   CONVERTIBLE      HARMONIC      VARIABLE    ANISOTROPY  NON-WETTABLE

    HYD. COND. ALONG ROWS =   3.00000     FOR LAYER   1

   HORIZ. ANI. (COL./ROW) =   1.00000     FOR LAYER   1

  HORIZ. TO VERTICAL ANI. =   3.00000     FOR LAYER   1

 WEL -- WELL PACKAGE FOR NWT VERSION 1.1.4,  4/01/2018 INPUT READ FROM UNIT   20
 # WEL package for  MODFLOW-NWT, generated by Flopy.
 No named parameters
 MAXIMUM OF      3 ACTIVE WELLS AT ONE TIME
 No named parameters
  PHIRAMP WILL BE SET TO A DEFAULT VALUE OF 1.0E-5
  WELLS WITH REDUCED PUMPING WILL BE REPORTED TO THE MAIN LISTING FILE


     0 Well parameters

 SFR7 -- STREAMFLOW ROUTING PACKAGE, VERSION 1.0.5, 2012-04-05
         INPUT READ FROM UNIT  14
 # SFR package for  MODFLOW-NWT, generated by Flopy.
 No named parameters


 NUMBER OF STREAM REACHES IS    9

 NUMBER OF STREAM SEGMENTS IS    1

 NUMBER OF STREAM PARAMETERS IS    0

 NUMBER OF STREAM SEGMENTS DEFINED USING PARAMETERS IS              0

 MAXIMUM ERROR FOR STREAM LEAKAGE RATES IS  1.00E-04

 CONSTANT FOR MANNINGS EQUATION IS  8.6400E+04



 FLOW TO AND FROM GROUND WATER FOR EACH STREAM REACH WILL BE SAVED ON UNIT   40


   STREAM NETWORK DESCRIPTION: 

   LAYER    ROW    COL   SEGMENT   REACH     LENGTH
                          NUMBER   NUMBER    IN CELL        
   --------------------------------------------------
      1      4      3       1        1    3.7300E+02
      1      4      4       1        2    2.9017E+02
      1      5      4       1        3    2.8024E+02
      1      5      5       1        4    5.7739E+02
      1      5      6       1        5    1.4008E+02
      1      6      6       1        6    4.4751E+02
      1      6      7       1        7    4.0260E+02
      1      7      7       1        8    1.8499E+02
      1      7      8       1        9    3.2002E+02



1
                            STRESS PERIOD NO.    1, LENGTH =   1.000000    
                            -----------------------------------------------

                              NUMBER OF TIME STEPS =     1

                               MULTIPLIER FOR DELT =     1.000

                            INITIAL TIME STEP SIZE =   1.000000    

 WELL NO.  LAYER   ROW   COL   STRESS RATE         
 --------------------------------------------------
      1      1      4      4      -1000.    
      2      1      5      6      -5000.    
      3      1      5      6      -2000.    

      3 WELLS



 CLASSIFICATION & COUNT OF STREAM SEGMENTS BASED ON SOURCE OF INFLOW:

                HEADWATER     DIVERSION     RECEIVES TRIBUTARY FLOW
                ---------     ---------     -----------------------
                     1              0               0



                    INPUT DATA FOR EACH STREAM SEGMENT
 ---------------------------------------------------------------------------------------------

 SEGMENT    SEG.     INFLOW   OVERLAND   STREAM    STREAM   ICALC  OUTFLOW  DIVERSION PRIORITY
    NO.    LENGTH     RATE     RUNOFF      ET       PPT.    METH.  TO SEG.  FROM SEG.    NO.

      1  3.016E+03 1.000E+01 0.000E+00 0.000E+00 0.000E+00    1        0        0        0


         STREAMBED PROPERTIES AND STREAM DIMENSIONS

 SEGMENT     BED HYD. COND.      BED THICKNESS     ELEV.-TOP OF BED     WIDTH OF STREAM     DEPTH OF STREAM    STREAM ROUGHNESS
    No.     UPPER     LOWER     UPPER     LOWER     UPPER     LOWER     UPPER     LOWER     UPPER     LOWER   CHANNEL      BANK

     1  1.000E+00 1.000E+00 1.000E+00 1.000E+00 9.500E+00 9.400E+00 1.000E-01 1.000E-01                     1.000E-01




 SOLVING FOR HEAD 

    ------------------------------------------------
       NWT REQUIRED           32 OUTER ITERATIONS 
       AND A TOTAL OF        207 INNER ITERATIONS.
    ------------------------------------------------

 OUTPUT CONTROL FOR STRESS PERIOD    1   TIME STEP    1
    PRINT BUDGET
    SAVE HEAD FOR ALL LAYERS
    SAVE BUDGET
 UBUDSV SAVING "   CONSTANT HEAD" ON UNIT 40 AT TIME STEP  1, STRESS PERIOD    1
 UBUDSV SAVING "FLOW RIGHT FACE " ON UNIT 40 AT TIME STEP  1, STRESS PERIOD    1
 UBUDSV SAVING "FLOW FRONT FACE " ON UNIT 40 AT TIME STEP  1, STRESS PERIOD    1

 WELLS WITH REDUCED PUMPING FOR STRESS PERIOD     1 TIME STEP     1
   LAY   ROW   COL         APPL.Q          ACT.Q        GW-HEAD       CELL-BOT
     1     4     4  -0.100000E+04  -0.191758E+03   0.280382E+00   0.000000E+00
     1     5     6  -0.500000E+04  -0.116599E+03   0.909677E-01   0.000000E+00
     1     5     6  -0.200000E+04  -0.466397E+02   0.909677E-01   0.000000E+00




 UBUDSV SAVING "  STREAM LEAKAGE" ON UNIT 40 AT TIME STEP  1, STRESS PERIOD    1
 UBUDSV SAVING "STREAMFLOW OUT  " ON UNIT 40 AT TIME STEP  1, STRESS PERIOD    1

 HEAD WILL BE SAVED ON UNIT    3 AT END OF TIME STEP    1, STRESS PERIOD    1
1
  VOLUMETRIC BUDGET FOR ENTIRE MODEL AT END OF TIME STEP    1, STRESS PERIOD   1
  ------------------------------------------------------------------------------

     CUMULATIVE VOLUMES      L**3       RATES FOR THIS TIME STEP      L**3/T
     ------------------                 ------------------------

           IN:                                      IN:
           ---                                      ---
             STORAGE =           0.0000               STORAGE =           0.0000
       CONSTANT HEAD =         344.9944         CONSTANT HEAD =         344.9944
               WELLS =           0.0000                 WELLS =           0.0000
      STREAM LEAKAGE =          10.0000        STREAM LEAKAGE =          10.0000

            TOTAL IN =         354.9944              TOTAL IN =         354.9944

          OUT:                                     OUT:
          ----                                     ----
             STORAGE =           0.0000               STORAGE =           0.0000
       CONSTANT HEAD =           0.0000         CONSTANT HEAD =           0.0000
               WELLS =         354.9966                 WELLS =         354.9966
      STREAM LEAKAGE =           0.0000        STREAM LEAKAGE =           0.0000

           TOTAL OUT =         354.9966             TOTAL OUT =         354.9966

            IN - OUT =      -2.1667E-03              IN - OUT =      -2.1667E-03

 PERCENT DISCREPANCY =          -0.00     PERCENT DISCREPANCY =          -0.00






         TIME SUMMARY AT END OF TIME STEP    1 IN STRESS PERIOD    1
                    SECONDS     MINUTES      HOURS       DAYS        YEARS
                    -----------------------------------------------------------
   TIME STEP LENGTH  86400.      1440.0      24.000      1.0000     2.73785E-03
 STRESS PERIOD TIME  86400.      1440.0      24.000      1.0000     2.73785E-03
         TOTAL TIME  86400.      1440.0      24.000      1.0000     2.73785E-03
1

 Run end date and time (yyyy/mm/dd hh:mm:ss): 2019/09/12 20:30:30
 Elapsed run time:  0.003 Seconds

